monkey/monkey_island/cc/setup/certificate/certificate_setup.py

```diff
@@ -0,0 +1,27 @@
+import logging
```

Comment:
Since there are no other files in `cc/setup/certificate`, can we move this file to `cc/setup/certificate_setup.py`?